libraries/rush-lib/src/logic/RushConstants.ts

```diff
@@ -91,6 +91,13 @@ export class RushConstants {
    */
   public static readonly pnpmfileV6Filename: string = '.pnpmfile.cjs';
 
+  /**
+   * The folder name used to store patch files for pnpm
+   * Example: `C:\MyRepo\common\config\pnpm\patches`
```

Comment:
I'm thinking it would be better to call this folder `common\config\pnpm-patches` similar to `common\config\git-hooks`.  Introducing additional levels of nesting is advantageous only if we expect those folders to contain many things.